packages/editor-ui/src/components/NodeDetailsView.vue

```diff
@@ -338,8 +338,8 @@ export default mixins(
 		},
 	},
 	watch: {
-		activeNode(node, oldNode) {
-			if (node && !oldNode && !this.isActiveStickyNode) {
```

Comment:
@mutdmour do you think it is safe to remove this condition from the watcher? by this, we get an immediate proper update of the components and I'm just guessing this was for external hook invoking and/or telemetry call (so they run only when a user opens the node details view, but now everything is running also when changing node by clicking on the link in the error message)

Reply:
i think it's fine..

Reply:
but test it well :)